src/scripts/main.js

```diff
@@ -5,7 +5,7 @@ const listLi = [...document.querySelectorAll('[data-salary')];
 
 function sortList(list) {
   const replace = (str) => {
-    return +str.slice(1).split(',').join('');
```

Comment:
you should replace for empty string, not `'.'`
`138,575` - its 138 thousands and 575,  after replace `138.575`, it's just 138 and decimals, so your sorting isn't working correct